cola/jax_fns.py

```diff
@@ -96,7 +92,7 @@ def eig(A):
     return jax.device_put(w, device), jax.device_put(v, device)
 
 
-def eye(n, m=None, dtype=None, device=None):
+def eye(n, m, dtype, device):
```

Comment:
@mfinzi, keep an eye on this change. I chose to make m not optional to avoid having the following signature `def eye(n, dtype, device, m=None)` as that might create confusion and unexpected behavior